Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -186,6 +187,49 @@ internal set
         // keep track of all sceneIds to detect scene duplicates
         static readonly Dictionary<ulong, NetworkIdentity> sceneIds = new Dictionary<ulong, NetworkIdentity>();
 
+
+
+        /// <summary>
+        /// This is invoked on clients when the server has caused this object to be destroyed.
+        /// <para>This can be used as a hook to invoke effects or do client specific cleanup.</para>
+        /// </summary>
+        ///<summary>Called on clients when the server destroys the GameObject.</summary>
+        public UnityEvent OnNetworkDestroy = new UnityEvent();
```

Comment:
Does this show in the inspector?  IIRC, we need to add a [SerializeField]

Reply:
Confirmed via screenshot in discord to be working properly in inspector. resolving.